lib/jellyfish/room.ex

```diff
@@ -72,6 +72,11 @@ defmodule Jellyfish.Room do
     end
   end
 
+  @spec request_forwarded_tracks_number_in_room(id()) :: :ok
+  def request_forwarded_tracks_number_in_room(room_id) do
+    GenServer.cast(registry_id(room_id), {:forwarded_tracks_number, self()})
```

Comment:
Same thing as with the function name in the engine -- but that's a nitpick

Reply:
I would call this function the same way it's called in rtc engine